uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-balance-tab.component.js

```diff
@@ -148,9 +148,9 @@ define([
      *
      * @param {Object} event - the component event handler.
      * @param {Object} _filters_ - The filter values to use for updating the report.
-     * @param {Object} _filters_.absence_period - the absence period id to filter by.
-     * @param {Object} _filters_.absence_type - the abence type id to filter by.
-     * @param {Object} _filters_.managed_by - the ID of the manager that is retrieving the report.
+     * @param {String} _filters_.absence_period - the absence period id to filter by.
```

Comment:
I think @AkA84 meant that if there are 2 params, there must be also 2 `@param` descriptions. If you'd like to describe the object, please feel free to use next lines + indentation for the description:

```js
* @param {Object} _filters_ - filters object containing the following params:
*   absence_period - number, the absence period id to filter by
*   absence_type - ...
*   managed_by  - ...
```